src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/Microsoft.DotNet.UpgradeAssistant.Abstractions.csproj

```diff
@@ -28,6 +29,15 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Abstractions" />
     <PackageReference Include="Microsoft.Extensions.FileProviders.Abstractions" />
     <PackageReference Include="System.Linq.Async" />
+    <PackageReference Include="System.Text.Json" />
   </ItemGroup>
 
+  <Target Name="PrivateAssetsIfExtension" Condition=" '$(IsExtension)' == 'true' " BeforeTargets="CollectPackageReferences">
```

Comment:
Is this updating extensions to not carry NuGet-referenced libraries with them? I would think they might need those assets at runtime if UA doesn't have the same NuGet dependencies.

Reply:
Not needed anymore. It made the abstractions libraries local as they'll be in the main output